components/style/properties/properties.mako.rs

```diff
@@ -1141,7 +1141,10 @@ impl PropertyDeclaration {
                   matches!(*value, DeclaredValue::WithVariables(_))
               },
           % endfor
-          PropertyDeclaration::Custom(..) => true
+          PropertyDeclaration::Custom(_, ref value) => match *value {
```

Comment:
nit: You can use `!matches!(*value, DeclaredValue::CSSWideKeyword(..))` instead of the match, which is a bit shorter.